src/ConnectedKubernetes/custom/New-AzConnectedKubernetes.ps1

```diff
@@ -230,7 +268,11 @@ function New-AzConnectedKubernetes {
         Set-Item -Path Env:HELM_EXPERIMENTAL_OCI -Value 1
         #Region pull helm chart
         try {
-            helm chart pull $RegisteryPath --kubeconfig $KubeConfig --kube-context $KubeContext
+             if ($HelmVersion -ge "v3.7") {
```

Comment:
Comparing versions as strings is dangerous. Consider if $HelmVersion = "v3.10", the result would be false, but it should be true.

I recommend use .NET type System.Version, for example:
```powershell
[System.Version]"3.10" -gt [System.Version]"3.7"
```

Reply:
Looks great!

Reply:
Done.